src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -7,6 +7,9 @@
     position: fixed;
     right: 0;
     z-index: 99999;
+    -webkit-border-top-left-radius: 4px;
```

Comment:
Same comment for the `-webkit-` prefix: it seems to be unneeded since Chrome 4.0 and Safari 5.0.